packages/svelte-ux/src/routes/docs/components/SelectField/+page.svelte

```diff
@@ -102,6 +107,25 @@
   <SelectField {options} loading />
 </Preview>
 
+<h2>Async options</h2>
+
+<Preview>
+  <SelectField options={optionsAsync} {loading} />
+  <Button
+    on:click={() => {
+      loading = true;
+      // simulate async loading with a loading of 2 seconds!
+      setTimeout(() => {
+        optionsAsync = optionsAsync.length === 0 ? options : [];
+        loading = false;
+      }, 2000);
```

Comment:
Should unloading happen immediately?

Reply:
It's an example... So it doesn't really matter. Butt ok 😜